### DISCUSSION TITLE
Adding new language under "Translation"

### DISCUSSION THREAD

**Discussion by Zyles** [directus/directus#2732] [Archive]
When adding a new translation for a field. What procedure needs to be done to add a new language here?

**Comment by alexvdvalk** [comment]
https://docs.directus.io/configuration/translation-strings.html#translation-strings

and

https://docs.directus.io/contributing/translations.html#app-translations

**Reply by Zyles** [reply]
> https://docs.directus.io/configuration/translation-strings.html#translation-strings
> 
> and
> 
> https://docs.directus.io/contributing/translations.html#app-translations

That is not what I am asking.

**Reply by br41nslug** [reply]
What type of field translations do you mean? 
Presumably the Field Label translations which are connected to the built-in languages managed in Crowdin (covered in the second link https://docs.directus.io/contributing/translations.html#requesting-a-new-language ). 
If you meant the Translations Field itself that will create a "Languages" Collection you can manage manually.

**Reply by Zyles** [reply]
I want to support two languages.

Field "color" should have options in two languages.

Or field "title" should have title in two languages.
